_posts/2024-02-23-gsoc2024.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title: Google Summer of Code 2024
+---
+
+<p>
+<img
+src="https://developers.google.com/open-source/gsoc/resources/downloads/GSoC-Vertical.svg"
+title="Google Summer of Code 2024" alt="Google Summer of Code with
+MDAnalysis 2024"
+style="float: right; height: 10em; " />
+</p>
+
+MDAnalysis has been accepted as an [organization][org] for [Google Summer of
+Code 2024][gsoc]! If you are interested in working with us this summer and you
+are new to open source, please read the advice and links below and write to us
+on the [GSoC with MDAnalysis discussion forum][discussion forum].
+
+We are looking forward to all applications from *any new and beginner
+open source contributors over 18 years old* or *students*. Projects
```

Comment:
Students also need to be 18 or over. Worth linking to the [eligibility criteria](https://buildyourfuture.withgoogle.com/programs/summer-of-code)?

Reply:
Done.